===================================================================
RCS file: /cvsroot/djvu/djvulibre-3.5/libdjvu/GString.h,v
retrieving revision 1.19
retrieving revision 1.20
diff -u -r1.19 -r1.20
--- djvulibre-3.5/libdjvu/GString.h	2004/08/06 15:11:29	1.19
+++ djvulibre-3.5/libdjvu/GString.h	2005/07/22 15:28:07	1.20
@@ -138,6 +138,10 @@
 # endif
 #endif
 
+class GBaseString;
+class GUTF8String;
+class GNativeString;
+
 // Internal string representation.
 class GStringRep : public GPEnabled
 {
@@ -433,8 +437,6 @@
   virtual unsigned long getValidUCS4(const char *&source) const;
 };
 
-class GUTF8String;
-class GNativeString;
 
 /** General purpose character string.
     Each dirivied instance of class #GBaseString# represents a
